test/videojs.ads.test.js

```diff
@@ -709,17 +708,14 @@ test('adscanceled allows us to transition from content-set to content-playback',
 
 test('adscanceled allows us to transition from ads-ready? to content-playback', function() {
   var callback;
-  expect(6);
+  expect(5);
```

Comment:
window.clearImmediate was being called twice causing the assert inside of it to be executed twice. When running with the code coverage tool QUnit was throwing errors because of the asserts within these timeout overrides. I moved the assert out of the functions and therefore only needed to be one.  I can add back a check that clearImmediate is actually being called with player.ads.cancelPlayTimeout to more closely resemble the initial, but there will still be one less assert.